### PR TITLE
Formatting

### PR DESCRIPTION
**Pull request by tsriram**
Looks like a bunch of files are not formatted on `master`. All these changes are done by running `yarn format`.

We should probably look at enforcing formatting on every commit or something like that!

### CODE REVIEW

**Comment by KyleAMathews** [comment]
Deploy preview for *using-postcss-sass* failed.

Built with commit ae51776c2a1d9f0c616af8298ccd6a531db66c99

https://app.netlify.com/sites/using-postcss-sass/deploys/5b742f02c6aed667d53e1ed7

**Comment by KyleAMathews** [comment]
Oh, looks like this broke some snapshots? Could you run `yarn test:update` locally to see if that fixes things?

File: packages/gatsby-codemods/src/transforms/__testfixtures__/global-graphql-calls/import-default.output.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /* eslint-disable */
-import Gatsby, { graphql } from "gatsby";
+import Gatsby, { graphql } from "gatsby"
```
Comment (m-allanson) [inline packages/gatsby-codemods/src/transforms/__testfixtures__/global-graphql-calls/import-default.output.js]: Hmm maybe we should have a `.prettierignore` that skips anything in `__testfixtures__`

**Comment by KyleAMathews** [comment]
Deploy preview for *gatsbyjs* failed.

Built with commit ae51776c2a1d9f0c616af8298ccd6a531db66c99

https://app.netlify.com/sites/gatsbyjs/deploys/5b742f01c6aed667d53e1ecb

File: packages/gatsby-codemods/src/transforms/__testfixtures__/global-graphql-calls/import-default.output.js (hunk 1 above)
Comment (tsriram) [reply packages/gatsby-codemods/src/transforms/__testfixtures__/global-graphql-calls/import-default.output.js]: I tried doing this but it seems to be breaking more tests or I'm doing something wrong :smile: 

Will try to get them fixed tomorrow.

**Comment by tsriram** [comment]
Before trying to fix tests in my branch, I tried running tests on `master` and I see quite a few tests are failing there too. But the builds are passing in Travis for other PRs! Any idea why this could happen? I've tried clearing Jest cache but it doesn't help.

**Comment by tsriram** [comment]
Well, I could not fix the tests locally by any means. Had to delete the repo, clone it again and all tests started working :man_shrugging: Have rebased my branch with `master` and ran `yarn format` again to fix the formatting of files in current `master`.

**Comment by m-allanson** [comment]
How strange... looks like it's working now though. Thanks @tsriram 😄 🙏

**Pull request merged**